javalib/src/main/scala/java/lang/System.scala

```diff
@@ -24,13 +24,15 @@ object System {
   def identityHashCode(x: Object): scala.Int =
     x.cast[Word].hashCode
 
-  def getenv(name: String): String                      = ???
   def clearProperty(key: String): String                = ???
   def getProperties(): Properties                       = ???
   def getProperty(key: String): String                  = ???
   def getProperty(key: String, default: String): String = ???
   def setProperty(key: String, value: String): String   = ???
 
+  def getenv(): Map[String, String] = envVars
+  def getenv(key: String): String   = envVars.get(key)
```

Comment:
I'd say `envVars.get` is fine the way it is. Micro-optimising things is usually not a good idea.